tools/extract_baserom.py

```diff
@@ -56,12 +56,17 @@ def main():
 
     args.output_dir.mkdir(parents=True, exist_ok=True)
     for dma_name, dma_entry in zip(dma_names, dma_entries):
-        if dma_entry.is_compressed():
+        if dma_entry.is_syms():
+            if (dma_entry.vrom_end - dma_entry.vrom_start) == 0:
+                continue
```

Comment:
why do we not want to extract an empty file in this case?

Reply:
For some reason I thought it was needed for matching in MM, but it is not. I can remove it.